packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -303,6 +304,11 @@ let statusColumn = new TableColumn<ContainerInfoUI | ContainerGroupInfoUI>('Stat
   align: 'center',
   width: '70px',
   renderer: ContainerColumnStatus,
+  comparator: (a, b) => {
+    const bStatus = ('status' in b ? b.status : 'state' in b ? b.state : '') ?? '';
```

Comment:
Can't it be rewrittten as bStatus = b.status ?? b.state ?? ''; ?

Reply:
@jeffmaury It gives me an error saying `Property 'status' does not exist on type 'ContainerGroupInfoUI | ContainerInfoUI'.
  Property 'status' does not exist on type 'ContainerInfoUI'` and `Property 'state' does not exist on type 'ContainerGroupInfoUI | ContainerInfoUI'.
  Property 'state' does not exist on type 'ContainerGroupInfoUI'`